backend/app/rest/user_routes.py

```diff
@@ -94,19 +95,46 @@ def get_users():
                     500,
                 )
 
+@blueprint.route("/user-status", methods=["GET"], strict_slashes=False)
+@require_authorization_by_role({"Relief Staff", "Regular Staff", "Admin"})
```

Comment:
we can't use require_auth_by_role here because this endpoint will just throw a 401 if the user is not logged in and is trying to sign up

Reply:
the only issue I see with removing this is that not having it will give information about users that exist on the platform (for example, if an outsider were to input an email of a user on the platform, it would tell us if they have an existing account). one way to avoid this is to create invite links so they can't be shared/accessed from outside

Reply:
I can make the invite links into a ticket so you don't need to address it with this pr

Reply:
yeah, that makes sense, i think that it should be easy to send an email when the user gets invited